lk2nd/smp/cpu-boot.c

```diff
@@ -56,6 +61,57 @@ int qcom_set_boot_addr(uint32_t addr, bool arm64)
 				QCOM_SCM_BOOT_FLAG_COLD_ALL, addr);
 }
 
+void qcom_power_on_l2_cache(uint32_t mpidr, uint32_t base)
```

Comment:
Maybe
```suggestion
void qcom_power_up_l2_cache(uint32_t mpidr, uint32_t base)
```
to be consistent with other functions in this file?

Reply:
Sure np, makes sense to me